cucumber-junit-platform-engine/README.md

```diff
@@ -266,9 +266,39 @@ Feature: Isolated scenarios
 
 with this configuration:
 
+
 ```properties
 cucumber.execution.exclusive-resources.isolated.read-write=org.junit.platform.engine.support.hierarchical.ExclusiveResource.GLOBAL_KEY
 ```
+### Sequential Scenario Execution With Parallel Run
+
+By default, when the configuration parameter `cucumber.execution.parallel.enabled` set to `true`, all applicable
```

Comment:
Consider simplifying this phrase and its duplicates to "when parallel execution is enabled".